core/src/main/java/org/frankframework/receivers/PullingListenerContainer.java

```diff
@@ -392,27 +392,14 @@ public void run() {
 			}
 		}
 
-		private void rollBack(TransactionStatus txStatus, RawMessageWrapper<M> rawMessage, String reason) {
+		private void rollBack(TransactionStatus txStatus, String reason) {
 			if (log.isDebugEnabled()) {
 				String stackTrace = Arrays.stream(Thread.currentThread().getStackTrace())
 					.map(StackTraceElement::toString)
 					.reduce("\n", (acc, element) -> acc + "    at " + element + "\n");
 				log.debug("Rolling back TX, reason: {}, stack:{}", reason, stackTrace);
 			}
-			try {
-				txManager.rollback(txStatus);
-			} finally {
-				if (inProcessStateManager!=null) {
```

Comment:
Missen we niet iets als dit finally block verdwijnt?

Reply:
Wat hier wordt gedaan is het record weer in "Available" zetten -- maar dat wordt later ook gedaan, buiten deze method om.

Ik controleer de hele flow nog een keer of er een mogelijkheid is, dat dit verderop eventueel overgeslagen zou kunnen worden als er fouten optreden, maar voor zover ik zie zit het goed.

Reply:
Aha, ok, dat wist ik niet!